Bottom: 1a579fa7cbed51f78d516f1d918a502590e24973
Top:    60230df43fea47a917e663b9de20e7bfa04d6f90
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-05-24 17:16:32 +0800

Refresh of vhost-device-iotlb-api

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index fcd81d7..7dd90f8 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -28,7 +28,6 @@
 #include <linux/module.h>
 #include <linux/sort.h>
 #include <linux/interval_tree_generic.h>
-#include <linux/ktime.h>
 
 #include "vhost.h"
